zipline/data/bundles/quandl.py

```diff
@@ -105,6 +106,14 @@ def fetch_data_table(api_key,
                 show_progress=show_progress,
             )
 
+        except HTTPError as err:
+            if err.code == 400:
```

Comment:
Can we just check this condition when we read the QUANDL_API_KEY out of the environment on line 210?